packages/plugins/src/pipeline.ts

```diff
@@ -124,13 +123,13 @@ class Pipeline {
     });
 
     // Check if the plugins should be initialized
-    if (this.properties.initialize) this.initialize();
+    if (this.properties.initialize) void this.initialize(); // TODO: Await this
```

Comment:
This will probably need to be fixed as it no longer waits for the initialization to finish before starting the server.